packages/generator/src/commonTemplateHelper.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Given an individual type or an array of types in the format Array\<Foo | Baa\>
+ * will return either the type prefixed by the namespace, or the Array with each type prefixed
+ * eg. Array\<types.Foo | types.Baa\>
+ *
+ * @param content - to be parsed for types to prefix with a namespace
+ * @param namespace - to be prefixed to types
+ * @returns the content prefixed with the namespace
+ */
+export function addNamespace(content: string, namespace: string): string {
+  // Not handling invalid content.
+  if (!content) {
+    throw new Error("Invalid content");
+  }
+  // Not handling invalid namespace.
+  if (!namespace) {
+    throw new Error("Invalid namespace");
+  }
+
+  // if the content is an array, extract all of the elements
+  let types = content;
+  let arrayType = false;
+  if (content.match("^Array") !== null) {
+    types = content.replace(/Array|<|>/g, "");
+    arrayType = true;
+  }
+  // Get a handle on individual types
+  const typesToProcess = types.split("|");
+  const namespaceTypes: string[] = [];
+
+  // for each type
+  typesToProcess.forEach(checkType => {
+    // trim the fat
+    const actualType = checkType.trim();
+    // check if there's an actual type present
+    if (actualType === "") {
+      throw new Error("Empty type found");
+    }
+
+    // void and object types don't get a namespace
+    if (["void", "object"].includes(actualType.toLocaleLowerCase())) {
```

Comment:
```suggestion
    if (["void", "object"].includes(actualType)) {
```
Converting to lowercase is unnecessary. Types are case-sensitive. (There technically is an `Object` type, but we shouldn't ever encounter it.)

Reply:
We did encounter object and Object in the list...which is why this was added.

Reply:
In that case, our usage of `Object` should be removed. Using `Object` the #1 thing on TypeScript's official [list of do's and don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html).

We use `Object` in a single location: https://github.com/SalesforceCommerceCloud/commerce-sdk/blob/0e9a36810365b31d2f115bb37d289e1b1d78107e/packages/generator/src/templateHelpers.ts#L94

Changing it to return `object` instead will bring us in line with best practices and will make the SDK more consistent.

The one caveat is that `object` will complain about property access, but `Object` will not. But based on our other usage of `object`, this doesn't seem to be a major concern.